fastparse/shared/src/main/scala/fastparse/Api.scala

```diff
@@ -112,28 +112,31 @@ class ByteApi() extends Api[Byte, Array[Byte]]() {
   }
 }
 
-object allByte extends ByteApi{
+object byte extends ByteApi{
   implicit def parserApi[T, V](p: T)
                               (implicit c: T => core.Parser[V, Byte, Array[Byte]]): ParserApi[V, Byte, Array[Byte]] =
     new ParserApiImpl[V, Byte, Array[Byte]](p)
 
-  val AnyWord = P( AnyByte.rep(exactly=2) )
-  val AnyDword = P( AnyByte.rep(exactly=4) )
+  val Word16 = P( AnyByte.rep(exactly=2) )
+  val Word32 = P( AnyByte.rep(exactly=4) )
+  val Word64 = P( AnyByte.rep(exactly=8) )
 }
-object emptyByteApi extends ByteApi
+object byteNoApi extends ByteApi
 
 object ByteUtils {
 
-  import allByte._
+  import byte._
 
   trait ByteFormat {
     def wrapByteBuffer(byteSeq: ByteSeq): ByteBuffer
 
-    val AnyWordI = P(AnyWord.!).map(wrapByteBuffer(_).getShort & 0xffff)
-    val AnyDwordI = P(AnyDword.!).map(wrapByteBuffer(_).getInt)
+    val Int16 = P(Word16.!).map(wrapByteBuffer(_).getShort & 0xffff)
```

Comment:
& 0xffff makes short value unsigned by converting it to the `Int`, so it can be slightly confusing to have `Int16` returning unsigned value in `Int` and `Int32` returning signed int value.